rpcserver.go

```diff
@@ -4149,6 +4152,7 @@ func (r *rpcServer) ListPayments(ctx context.Context,
 			Path:            path,
 			Fee:             int64(route.TotalFees().ToSatoshis()),
 			PaymentPreimage: hex.EncodeToString(preimage[:]),
+			PaymentRequest:  string(payment.Info.PaymentRequest),
```

Comment:
I think this is okay, but just to be sure: does this work if the `PaymentRequest` is nil?

Reply:
looks okay: https://play.golang.org/p/N3ss9fZDYpr